magento/pwa-studio
# Peregrine implementation/receipt into checkout slice

## Description

The `checkoutReceipt` slice doesn't really need to exist separate from `checkout`.

## Related Issue





Closes #1685 .

## Verification Steps

1. Work through checkout and make sure the receipt opens.
2. On the receipt page, click the X, and then reopen the cart. It should show the empty cart view.
3. Work through checkout again and this time click 'create account' on receipt. It should navigate to the create account page. Reopen the cart, it should show an empty cart.
4. Work through checkout again and this time click the mask (left of the cart). It should close the cart. Reopen the cart and it should show the empty cart.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
